p4-16/spec/P4-16-spec.mdk

```diff
@@ -5523,6 +5524,42 @@ use to implement generalized switch statements.
 
 [GeneralizedSwitchStatements]: <https://github.com/p4lang/p4-spec/blob/master/p4-16/spec/docs/implementing-generalized-switch-statements.md>
 
+## Loop statement { #sec-loop-stmt }
+
+A `loop` statement executes a statement zero or more times, based on a condition
+or a range or collection.
+
+~ Begin P4Grammar
+
+[INCLUDE=grammar.mdk:loopStatement]
+
+[INCLUDE=grammar.mdk:forInitStatement]
+
+[INCLUDE=grammar.mdk:forUpdateStatement]
+
+~ End P4Grammar
+
+The basic 3-clause `for` statement is similar to a C for statement.  The init statements
```

Comment:
```suggestion
The basic 3-clause `for` statement is similar to a C `for` statement.  The init statements

Reply:
Thanks for the detailed comments.  Given that this PR has already been merged, it would be great if someone could create a new PR with the proposed changes.  vlstill I do not know if you are interested in doing so?

Reply:
I'll open a PR.